index.md

```diff
@@ -107,7 +109,9 @@ fun night of talks and camaraderie—beginners and experts welcome.
 {% include location-box.html venue=page.venue %}{% else %}
 {% include location-box-tbd.html %}{% endif %}
 
+{% if page.venue %}
```

Comment:
Also added this --- so the gratitude text doesn't show up when `venue` is not final. :p